services/api-docs/Dockerfile

```diff
@@ -1,19 +1,19 @@
 FROM node:12-alpine
+ARG NODE_ENV=production
 
 RUN apk add --no-cache bash git python make g++ bash
 
 COPY ./package.json /service/package.json
 COPY ./yarn.lock /service/yarn.lock
 
-RUN cd /service; yarn install --production=true
+RUN cd /service; yarn install --frozen-lockfile
```

Comment:
Does this also imply `--production=true`? We don't want devDependencies installed